scripts/microbe_stage/microbe_editor/microbe_editor.as

```diff
@@ -563,6 +578,26 @@ class MicrobeEditor{
     }
 
 
+    void getNucleusIsPresent()
+    {
+        double lengthMicrobe = double(editedMicrobe.length());
+        for(uint i = 0; i < editedMicrobe.length(); ++i){
+            auto organelle = cast<PlacedOrganelle>(editedMicrobe[i]);
+            auto name = organelle.organelle.name;
+            if (name=="nucleus"){
+                isNucleusPresent = true;
+                LOG_INFO("Nucleus is present!");
+                break;
+            }
+            else {
+                isNucleusPresent = false;
```

Comment:
Doesn't this line completely break what this method is attempting to do.

Reply:
if is not found variable is set to false, if found 1 time is set true and go out with break. of course if the function change and become with return value this set is going to die.

Reply:
Okay, I see the break now. It is still very weird to be setting it to false in the loop rather than setting it to false once.

Reply:
yhe actually i could just set it false before loop.